### PR TITLE
Feature/search

### PR DESCRIPTION
**Pull request by dhruv-divtech**

### CODE REVIEW

File: src/views/guestViews/guestLayout/Header.tsx [hunk 1]
```diff
@@ -132,7 +158,7 @@ const HeaderGuestComponent = () => {
               />
             </Box>
             {isMdUp && (
-              <Box display="flex" alignItems="center" gap={1} sx={{ cursor: 'pointer' }}>
+              <Box display="flex" alignItems="center" gap={1} sx={{ cursor: 'pointer' }} onClick={handleOpenFilterModal}>
```
Comment (WorkspaceChetan) [inline src/views/guestViews/guestLayout/Header.tsx:161]: make style component

File: src/views/guestViews/homePage/homeImageCards/index.tsx [hunk 2]
```diff
@@ -120,13 +141,35 @@ const HomeImageCard = ({
...
+              />
+            </PaginationMainBox>
           </ButtonMainBox>
+        ) : (
+          ''
         )}
+        {modelListing.length > 0
+          ? ''
+          : !isFavPage && (
+              <Box display="flex" justifyContent="center" alignItems="center" height="100%">
```
Comment (WorkspaceChetan) [inline src/views/guestViews/homePage/homeImageCards/index.tsx:169]: make style component

File: src/views/guestViews/searchPage/moreFilters/SliderFilter.tsx [hunk 3]
```diff
@@ -0,0 +1,36 @@
...
+  maxValue: number;
+  handleChange: (event: Event, newValue: number | number[]) => void;
+}) => (
+  <Slider
+    value={[fromValue, toValue]}
+    onChange={handleChange}
+    valueLabelDisplay="on"
+    min={minValue}
+    max={maxValue}
+    sx={{
```
Comment (WorkspaceChetan) [inline src/views/guestViews/searchPage/moreFilters/SliderFilter.tsx:22]: make style component

File: src/views/guestViews/searchPage/moreFilters/index.tsx [hunk 4]
```diff
@@ -0,0 +1,235 @@
...
+    setFilters(cloneDeep(initialFilters));
+    handleClose();
+  };
+
+  return (
+    <Drawer
+      anchor="right"
+      open={open}
+      onClose={handleClose}
+      sx={{
```
Comment (WorkspaceChetan) [inline src/views/guestViews/searchPage/moreFilters/index.tsx:154]: make style component

File: src/views/guestViews/guestLayout/Header.tsx (hunk 1 above)
Comment (dhruv-divtech) [reply src/views/guestViews/guestLayout/Header.tsx]: resolved

File: src/views/guestViews/homePage/homeImageCards/index.tsx (hunk 2 above)
Comment (dhruv-divtech) [reply src/views/guestViews/homePage/homeImageCards/index.tsx]: resolved

File: src/views/guestViews/searchPage/moreFilters/SliderFilter.tsx (hunk 3 above)
Comment (dhruv-divtech) [reply src/views/guestViews/searchPage/moreFilters/SliderFilter.tsx]: resolved

File: src/views/guestViews/searchPage/moreFilters/index.tsx (hunk 4 above)
Comment (dhruv-divtech) [reply src/views/guestViews/searchPage/moreFilters/index.tsx]: resolved

File: src/views/guestViews/searchPage/moreFilters/index.tsx [hunk 5]
```diff
@@ -0,0 +1,235 @@
...
+          scrollbarWidth: 'none',
+          width: '100%',
+          maxWidth: 528,
+          justifyContent: 'space-between',
+          backgroundColor: 'secondary.dark'
+        }
+      }}
+    >
+      <FilterHeader>
+        <Box sx={{ display: 'flex', gap: 1.5, alignItems: 'center' }}>
```
[the remark this answers is not in the document]
Comment (dhruv-divtech) [reply src/views/guestViews/searchPage/moreFilters/index.tsx]: resolved

**Pull request merged**